### DISCUSSION TITLE
Discussion for the biweekly meeting (2023/04/19)

### DISCUSSION THREAD

**Discussion by sczyh30** [opensergo/opensergo-specification#81] [General]
Discussion for the biweekly meeting of the OpenSergo & Sentinel community (2023/04/19)

---

本次 OpenSergo 双周会将于 2023/04/19（周三）与 Sentinel 社区周会合并进行，欢迎大家预留时间参与，也欢迎大家发起 topic 来分享。议程如下：

* 同步上一周期的进展，包括微服务治理 spec 建设、OpenSergo 统一控制面、社区生态接入，以及 Sentinel 2.0 治理能力演进进展。
* 社区分享与讨论
  * 统一控制面扩展性调研：[Envoy Gateway 扩展性](https://github.com/envoyproxy/gateway/blob/main/docs/latest/design/extending-envoy-gateway.md)
  * OpenSergo 流量防护与容错 spec 控制面预计算逻辑与数据面 proto 交互讨论
  * [中科院开源之夏 2023 事项](https://summer-ospp.ac.cn/)同步

主题：OpenSergo & Sentinel 社区双周会
时间：4月19日 周三 19:30 - 20:30
会议号：244 801 51010
电话呼入：+862281944261（中国大陆）+867388953916（中国大陆）
入会链接：https://meeting.dingtalk.com/j/S3a65zNaSFk

上次社区双周会记录：https://github.com/opensergo/opensergo-specification/discussions/79

**Comment by xijianyue** [comment]
收到！
